### ISSUE TITLE
Issue defining autoloads

### ISSUE THREAD

**Issue by Ergus** [jixiuf/vterm-toggle#24]
Hi:

After a fresh installation with use-package I am getting this message:

```
command-execute: Autoloading file /home/XXX/.emacs.d/elpa/vterm-toggle-20210130.1149/vterm-toggle.elc failed to define function vterm-toggle-cd
```

when I reinstall the package the problem seems to do away, but every time vterm is updated the problem comes back.

**Comment by jixiuf** [comment]
see https://emacs.stackexchange.com/questions/30898/how-to-solve-autoloading-failed-to-define-function-sml-mode

**Comment by Ergus** [comment]
> see https://emacs.stackexchange.com/questions/30898/how-to-solve-autoloading-failed-to-define-function-sml-mode

Hi I already checked that but that's not the problem. Any other idea?

**Comment by Ergus** [comment]
Looking at the vterm-toggle-autoloads.el file the code seems to be fine but the vterm-toggle.elc is wrong. It only has this inside:

```Elisp
(byte-code "\300\301!\210\300\302!\210\300\303!\210\300\304!\207" [require cl-lib tramp tramp-sh vterm] 2)
```

I had to recompile the file and then the generated code is fine. But after an update the regenerated code is wrong again.

**Comment by jixiuf** [comment]
should be fixed 6342d42e5c3016e62a622cf3c855fa9a33d6d24b

**Comment by Ergus** [comment]
thanks
